tests/core/test_decorators.py

```diff
@@ -31,7 +31,7 @@
     from typing_extensions import Literal  # type: ignore
 
 
-def test_check_function_decorators():
+def test_check_function_decorators() -> None:
```

Comment:
are these type annotations necessary? why type-annotate test functions?

Reply:
@cosmicBboy I added them to increase mypy coverage. Functions that have no annotations at all are not checked by mypy. Type checking our tests is useful because tests cover many real-life scenarios so we can figure out if our typing is correct and if it makes sense.